new/LIT7159PPHemamaS.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7159PPHemamaS" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Prayer against the evil eye for birth</title>
```

Comment:
The word ṭǝlā is probably related to ṣǝlā, which is probably related to ṣǝlā wagi, which is a demon. I would propose to change the title to "Prayer against the evil eye of Ṭǝlā in the moment of birth" or to "Prayer against the pain of birth by ʾayna ̣̣tǝlā".